Packs/AbuseDB/ReleaseNotes/1_0_36.md

```diff
@@ -0,0 +1,7 @@
+
+#### Integrations
+
+##### AbuseIPDB
+- Updated the Docker image to: *demisto/python3:3.12.11.4417419*.
+
+- Documentation and metadata improvements.
```

Comment:
Change the "Documentation and metadata improvements." to "Fixed an issue where the default value for the "days" argument in the "ip" command..."